README.md

```diff
@@ -42,7 +44,11 @@ var foods = [ ["apple","banana","strawberry"], ["pizza","fries","hamburger"] ];
 Create a For Loop that outputs the following string for each piece of fruit in the console. "I want to eat a [fruit]"
 
 ```js
-// write code here
+foods.forEach(function(fruits) {
+    fruits.forEach(function(fruit) {
+        console.log("I want to eat a " + fruit);
+    });
+});
```

Comment:
give this loop a try in the console, I'm seeing it print out both fruits and the other foods.. you actually only need one loop here, and that would be to loop through ```foods[0]```